app/models/planning_application.rb

```diff
@@ -347,6 +357,12 @@ def documents_for_decision_notice
     documents.for_display
   end
 
+  def allows_only_one_open_description_change
+    if open_description_change_requests.size > 1
```

Comment:
didn't `any?` work here?

Reply:
ah, this one is a leftover - we don't need it anymore since we've moved it - thanks, nice catch